packages/tasks/src/model-libraries.ts

```diff
@@ -784,6 +784,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	renderformer: {
+		prettyLabel: "Renderformer",
+		repoName: "Renderformer",
```

Comment:
```suggestion
		prettyLabel: "RenderFormer",
		repoName: "RenderFormer",
```

taken from 

<img width="713" height="171" alt="Image" src="https://github.com/user-attachments/assets/0d33a4ff-03a8-47b6-b9b9-fe858fd9feba" />